test/functional/apps/discover/group3/_doc_viewer.ts

```diff
@@ -61,7 +61,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
           return (await find.allByCssSelector('.kbnDocViewer__fieldName')).length === 4;
         });
 
-        await PageObjects.discover.findFieldByNameInDocViewer('.s');
+        await PageObjects.discover.findFieldByNameInDocViewer('.sr');
```

Comment:
that's because of the content of this test. 
We type 'geo' and we get 4 results.
The we type '.s' and before we were getting 2 results but now we get 4 because of matches from fuzzy search. I think it's not a good test to leave it like this because it might be accidential the number stays the same. So I prefer to instead type the string that changes the number of results

'first updates' part of the test:
<img width="545" alt="Screenshot 2024-07-10 at 12 46 44" src="https://github.com/elastic/kibana/assets/4283304/440ce546-2c9a-42cf-a69c-1a0c6e2cf350">

after typing .s:
<img width="542" alt="Screenshot 2024-07-10 at 12 43 17" src="https://github.com/elastic/kibana/assets/4283304/f4118552-274e-483b-8d49-504c8270f61c">


after typing .sr

<img width="528" alt="Screenshot 2024-07-10 at 12 46 52" src="https://github.com/elastic/kibana/assets/4283304/230046a8-bcdb-48df-a023-fc36bd29c146">

Reply:
Totally make sense, thanks for the explanation!